### PR TITLE
Alert users of error messages inside applications tab

### PR DESCRIPTION
**Pull request by afiune**
### :nut_and_bolt: Description
As an Automate user, when I use the Applications Tab,
I would like to be notified of any error happening in the backend,
So I can troubleshoot any problem on the system.

### :+1: Definition of Done
Display a notification alert at the top of the main frame inside the Applications tab of any error message that happens internally.

### :athletic_shoe: Demo Script / Repro Steps
- Enter the studio. `hab studio enter`
- Build and start the automate-ui in your preferred way. (mine: `start_automate_ui_background`)
- Start all services. `start_all_services`
- Navigate to the applications tab: https://a2-dev.test/applications

Quickly, kill the applications-service process and click in any filter in the UI. 

`pkill applications-service`

apps-tab-alert

### :chains: Related Resources
https://chefio.atlassian.net/browse/A2-884

### :white_check_mark: Checklist

- [ ] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [x] Code actually executed?
- [x] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

File: components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss [hunk 1]
```diff
@@ -22,6 +22,13 @@ app-page-picker {
   padding-bottom: 35px;
 }
 
+chef-alert {
+  -webkit-align-items: left;
+  align-items: left;
+  -webkit-justify-content: left;
+  justify-content: left;
```
Comment (scottopherson) [inline components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss]: `-webkit` prefixes shouldn't be needed.

**Comment by afiune** [comment]
Apparently, @msorens has unveiled a problem introduced with the change to use HTTP/2 in the load balancer: https://github.com/chef/automate/pull/661

Im verifying why or how this is happening and how to fix it. Thanks Michael! 💯

**Comment by afiune** [comment]
Fixed the problem, now the alert banner will print way better error messages coming from the backend! dope. 

Screen Shot 2019-06-25 at 8 36 53 PM

**Comment by susanev** [comment]
none of our other user notification banners have rounded corners like that, were those in the designs?

**Comment by afiune** [comment]
@susanev I am not sure I am modifying the alert in a way that the component changed to have rounded corners 🤔 but I will verify. Though, since you have more UI experience, can you point me where or what could it happen? Thank you

**Comment by apriofrost** [comment]
@susanev @afiune it seems that the default style of the `ChefAlert` component has the radius: https://ui-library.cd.chef.co/atoms/chef-alert I was trying to follow the pattern @susanev established in her design. Anything we can do to make it look the same (remove the radius)?

**Comment by susanev** [comment]
@apriofrost i think this is the one y'all want here... https://ui-library.cd.chef.co/molecules/chef-notification
i might be missing some context....

**Comment by apriofrost** [comment]
@susanev Ah this makes sense! I didn't realize we have similar ones. Just fount out ChefBanner under Molecules that looks a bit similar as well. Wish there could be more documents on when to use which. Maybe a topic to discuss with the UX folks.
@afiune Could you update the component to use the ChefNotification instead of ChefAlert? Sorry for the misguidance :)

**Comment by afiune** [comment]
@apriofrost @susanev On it! 

tenor-200099826

**Comment by afiune** [comment]
@susanev @apriofrost 
## Changed!

Screen Shot 2019-06-26 at 8 32 48 PM

**Comment by apriofrost** [comment]
@afiune looking great! thanks so much! @susanev Thanks for your attention to the details!

**Comment by srenatus** [comment]
👉 https://chefio.slack.com/archives/C61F9HHKK/p1561636054072300 In case anyone else is interested in what what changed on a protocol level between HTTP/1.1 and HTTP2.

**Pull request merged**